src/app/modules/group/components/manager-permission-dialog/manager-permission-dialog.component.html

```diff
@@ -25,50 +25,69 @@
         This panel lets you select what you allow the selected user or group of users to do on the current group and its descendants.
```

Comment:
**Bug** In the group managers page, both progress and switch are collapsible, it should not be the case

**Steps to reproduce**
1. Navigate to [this group managers](https://dev.algorea.org/branch/permissions-dialog-refact/en/#/groups/by-id/672913018859223173;path=52767158366271444/details/managers)
2. For 1st manager, click on the pen icon to edit manager permission
3. Click on the header of a progress or a switch